config.yaml

```diff
@@ -3,3 +3,15 @@ options:
     type: string
     default: internet
     description: Data Network Name (DNN)
+  amf-ip:
```

Comment:
Maybe `external-amf-ip` and `external-amf-hostname` would be better variable names. This would match with Prometheus' `external-url` [parameter](https://prometheus.io/docs/prometheus/latest/command-line/prometheus/).

Reply:
Done